ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -205,15 +205,13 @@ public struct AvatarGroup: View {
 
                 let ringPaddingInterspace = nextAvatarHasRing ? interspace - (ringOffset + ringOuterGap) : interspace - ringOffset
                 let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
-                let rtlRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOuterGap : -x + ringOffset)
-                let rtlNoRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOffset - ringOuterGap : -x)
                 let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
 
+                let cutoutSize = isLastDisplayed ? (ringOuterGap * 2) + imageSize : nextAvatarSize
                 let xPosition = currentAvatarHasRing ? x - ringOuterGap - ringOuterGap : x - ringOuterGap
-                let xPositionRTL = currentAvatarHasRing ? rtlRingPaddingInterspace : rtlNoRingPaddingInterspace
+                let xPositionRTL = -cutoutSize - interspace + ringOuterGap + (currentAvatarHasRing ? ringOffset : 0)
```

Comment:
seems little weird we would have to compute xpostitionrtl even thought we might not need it at all? can we optimize how we calculate xOrigin?

Reply:
@huwilkes how about initializing the xOrigin with a closure like below:

```
let xOrigin = {
    if layoutDirection == .rightToLeft {
        return <calculated value for RTL>
    }

    return <calculated value for LTR>
}()
```